### ISSUE TITLE
Update Imagenette to new version

### ISSUE THREAD

**Issue by Conchylicultor** [tensorflow/datasets#1679]
From https://github.com/tensorflow/datasets/pull/1261#issuecomment-601302108:

> the Imagenette dataset have changed:
> "The Imagenette and Imagewoof datasets have recently (Dec 6 2019) changed. They now have a 70/30 train/valid split." (from https://github.com/fastai/imagenette)
> The imagenette tensorflow dataset has still the small validation set.
> It is impossible to compare to the fastai leaderboard using this dataset.

**Comment by Eshan-Agarwal** [comment]
@Conchylicultor only new version should be kept or both ?

**Comment by Conchylicultor** [comment]
If possible, add config for the legacy versions, otherwise, it should be fine to only use the last version.

**Comment by Eshan-Agarwal** [comment]
@Conchylicultor  thanks for response adding config like this ?

```py
class OpenImagesV4(tfds.core.GeneratorBasedBuilder):

  BUILDER_CONFIGS = [
      OpenImagesV4Config(
          name='original',
          version=tfds.core.Version('0.2.0'),
          supported_version=[
            tfds.core.Version('1.0.0', "Major change in data"),
          ],
          description='Images at their original resolution and quality.'),
      ...
  ]

tfds.load('open_images_v4/original:1.*.*')
```

**Comment by Eshan-Agarwal** [comment]
@Conchylicultor  keeping last version , `imagewoof` dataset is not included in tensorflow. Also there is one `data request` for it #1317 and it have PR #1590, but in this only full-size data is generated 160px or 320px is not included. 
Can I include it or send PR for this ?
